### PR TITLE
BAU: Changes to deploy F2F in dev01/dev02.

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
Changes to be able to deploy F2F stub to dev01/dev02 env
 
### What changed
Done required change to f2f cf template

### Issue tracking

- [PYI-XXXX](https://govukverify.atlassian.net/browse/PYI-XXX)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

**Comment by Joe-Edwards-GDS** [comment]
Hi @rpayal - can you provide some background on this change? It doesn't look directly related to the F2F CIMIT work

**Comment by rpayal** [comment]
> Hi @rpayal - can you provide some background on this change? It doesn't look directly related to the F2F CIMIT work

Yeah thats why not link with ticket commits. This needed if u need to deploy f2f stub to ur local dev i.e either dev01/dev02 for testing f2f stub changes.

File: di-ipv-credential-issuer-stub/deploy/face-to-face/template.yaml
Comment (Joe-Edwards-GDS) [inline di-ipv-credential-issuer-stub/deploy/face-to-face/template.yaml:1]: Should we be making similar changes for other CRI stubs?

**Review by Joe-Edwards-GDS — APPROVED** [review]
I don't mind for you to merge this, but I'd quite like someone else (maybe @vam-gds ?) to see it.

File: di-ipv-credential-issuer-stub/deploy/face-to-face/template.yaml
Comment (rpayal) [reply di-ipv-credential-issuer-stub/deploy/face-to-face/template.yaml:1]: yeah that be good.
@vam-gds can u pls. review this, if needed will merge.

**Comment by rpayal** [comment]
Closing - done as part of https://github.com/govuk-one-login/ipv-stubs/pull/541

**Pull request closed without merging**